package-manifest.toml

```diff
@@ -4,173 +4,173 @@
-[[package.omicron-nexus.paths]]
-from = "smf/nexus"
-to = "/var/svc/manifest/site/nexus"
-[[package.omicron-nexus.paths]]
-from = "out/console-assets"
-to = "/var/nexus/static"
-# Note, we could just map the whole "out/certs" directory, but this ensures
-# both files exist.
-[[package.omicron-nexus.paths]]
-from = "out/certs/cert.pem"
-to = "/var/nexus/certs/cert.pem"
-[[package.omicron-nexus.paths]]
-from = "out/certs/key.pem"
-to = "/var/nexus/certs/key.pem"
-
 [package.oximeter-collector]
-rust.binary_names = ["oximeter"]
-rust.release = true
 service_name = "oximeter"
-zone = true
-[[package.oximeter-collector.paths]]
-from = "smf/oximeter"
-to = "/var/svc/manifest/site/oximeter"
+source.type = "local"
+source.rust.binary_names = ["oximeter"]
+source.rust.release = true
+source.paths = [ { from = "smf/oximeter", to = "/var/svc/manifest/site/oximeter" } ]
+output.type = "zone"
 
 [package.clickhouse]
 service_name = "clickhouse"
-zone = true
+source.type = "local"
+source.paths = [
+  { from = "out/clickhouse", to = "/opt/oxide/clickhouse" },
+  { from = "smf/clickhouse", to = "/var/svc/manifest/site/clickhouse" }
+]
+output.type = "zone"
 setup_hint = "Run `./tools/ci_download_clickhouse` to download the necessary binaries"
-[[package.clickhouse.paths]]
-from = "out/clickhouse"
-to = "/opt/oxide/clickhouse"
-[[package.clickhouse.paths]]
-from = "smf/clickhouse"
-to = "/var/svc/manifest/site/clickhouse"
 
 [package.cockroachdb]
 service_name = "cockroachdb"
-zone = true
+source.type = "local"
+source.paths = [
+  { from = "out/cockroachdb", to = "/opt/oxide/cockroachdb" },
+  { from = "common/src/sql", to = "/opt/oxide/cockroachdb/sql" },
+  { from = "smf/cockroachdb", to = "/var/svc/manifest/site/cockroachdb" },
+]
+output.type = "zone"
 setup_hint = "Run `./tools/ci_download_cockroachdb` to download the necessary binaries"
-[[package.cockroachdb.paths]]
-from = "out/cockroachdb"
-to = "/opt/oxide/cockroachdb"
-[[package.cockroachdb.paths]]
-from = "common/src/sql"
-to = "/opt/oxide/cockroachdb/sql"
-[[package.cockroachdb.paths]]
-from = "smf/cockroachdb"
-to = "/var/svc/manifest/site/cockroachdb"
 
 [package.internal-dns]
-rust.binary_names = ["dnsadm", "dns-server"]
-rust.release = true
 service_name = "internal-dns"
-zone = true
-[[package.internal-dns.paths]]
-from = "smf/internal-dns"
-to = "/var/svc/manifest/site/internal-dns"
+source.type = "local"
+source.rust.binary_names = ["dnsadm", "dns-server"]
+source.rust.release = true
+source.paths = [ { from = "smf/internal-dns", to = "/var/svc/manifest/site/internal-dns" } ]
+output.type = "zone"
 
 [package.omicron-gateway]
-rust.binary_names = ["mgs"]
-rust.release = true
 service_name = "mgs"
-zone = true
-[[package.omicron-gateway.paths]]
-from = "smf/mgs"
-to = "/var/svc/manifest/site/mgs"
+source.type = "local"
+source.rust.binary_names = ["mgs"]
+source.rust.release = true
+source.paths = [ { from = "smf/mgs", to = "/var/svc/manifest/site/mgs" } ]
+output.type = "zone"
```

Comment:
Should this have `output.intermediate_only = true`?

Reply:
Yeah, I accidentally added it to maghemite instead (wrong) instead of MGS (right). Fixed in ac12a68129620722e248a8258cbd4a02d8450d6d